frontend/app/src/scenes/Home/scenes/EditItem/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { RouteComponentProps, withRouter } from 'react-router';
+import {
+  Button,
+  Card,
+  Col,
+  Form,
+  Input,
+  notification,
+  Row,
+  Spin,
+  Typography,
+} from 'antd';
+import mainStyles from '../../styles.css';
+import styles from './styles.css';
+import { Item } from '../../interfaces';
+import axios, { AxiosResponse } from 'axios';
+import { handleApiError } from '../../../../services/util/errorHandler';
+import { AddItemPayload } from '../AddItem/interfaces';
+import { EditItemStateProps } from './interfaces';
+
+const { Title, Text } = Typography;
+
+class EditItem extends React.Component<
+  RouteComponentProps<any>,
+  EditItemStateProps
+> {
+  constructor(props: RouteComponentProps) {
+    super(props);
+    this.state = {
+      isLoading: false,
+      item: null,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchItem();
+  }
+
+  fetchItem = () => {
+    this.setState({ isLoading: true });
+    axios
+      .get(
+        window.location.origin +
+          '/core/academix/items/' +
+          this.props.match.params.id
+      )
+      .then((result: AxiosResponse<Item>) => {
+        if (result.status == 200) {
+          this.setState({ item: result.data, isLoading: false });
+        }
+      })
+      .catch((error) => {
+        handleApiError(
+          error,
+          'Something went wrong when trying to load the Item'
+        );
+        this.props.history.goBack();
+      });
+  };
+
+  onFinish = (values: any) => {
+    this.setState({ isLoading: true });
+    const item = {
+      link: values.link,
+      translations: [
+        {
+          name: values.name,
+          description: values.description,
+          language: 'en',
+        },
+      ],
+    };
+    axios
+      .put(
+        // eslint-disable-next-line max-len
+        `${window.location.origin}/core/academix/admin/items/${this.state.item?.id}/translations?isAllReset=false`,
+        item
```

Comment:
Do not override eslint rule here. split it into two

Reply:
sure